core/src/main/java/org/bitcoinj/quorums/InstantSendManager.java

```diff
@@ -463,8 +463,10 @@ boolean processPendingInstantSendLocks() throws BlockStoreException
             }
         }
 
-        if(context.masternodeSync.hasVerifyFlag(MasternodeSync.VERIFY_FLAGS.BLS_SIGNATURES))
+        if(context.masternodeSync.hasVerifyFlag(MasternodeSync.VERIFY_FLAGS.BLS_SIGNATURES) &&
+                !context.masternodeListManager.isQuorumRotationEnabled()) {
             batchVerifier.verify();
+        }
```

Comment:
We will only verify signatures if DIP24 is not active.  If it is active, we cannot verify signatures.

Reply:
Signatures that are without errors after this call are assumed to be verified.